sematic/db/tests/test_queries.py

```diff
@@ -101,6 +103,43 @@ def test_get_artifact(test_db, persisted_artifact: Artifact):  # noqa: F811
     assert artifact.json_summary == artifact.json_summary
 
 
+def test_save_artifact(test_db, test_storage):  # noqa: F811
+    artifact = make_artifact(42, int, storage=test_storage)
+    save_graph(artifacts=[artifact], runs=[], edges=[])
```

Comment:
The new `_save_artifact` function requires a `Session` parameter, and all the tests in this file only validate functions which create their own sessions, so I maintained the same usage pattern by employing the `save_graph` function here.